extension/content.js

```diff
@@ -263,6 +263,25 @@ function addPatchDiffLinks() {
 	`);
 }
 
+// Prompt user to confirm erasing a comment with the Cancel button
+$(document).on('click', function (event) {
+	// Check event.target instead of using a delegate, because Sprint doesn't support them
```

Comment:
Tried with `grunt custom:-ajax,-deprecated,-effects,-sizzle,-exports/amd,-dimensions,-event/alias,-offset`, but didn't help much. Still 49 KB. So I guess we can live with no event delegation. Can't drop `deferred` as the document ready event depends on it.

Alternatively, we could apply this tiny patch: https://github.com/bendc/sprint/pull/20/files ?